sympy/series/order.py

```diff
@@ -368,6 +368,14 @@ def contains(self, expr):
                 common_symbols = expr.variables
             if not common_symbols:
                 return None
+            if (self.expr.is_Pow and self.expr.base.is_Symbol
+                and self.expr.exp.is_positive):
+                if expr.expr.is_Pow and self.expr.base == expr.expr.base:
+                    return not (self.expr.exp-expr.expr.exp).is_positive
```

Comment:
Then this is leaving the Pow/Mul in the Add. OTOH if that is None you can't say it's definitely in the order, I think, so it seems ok. I didn't encounter failed tests.